libnopegl/src/distmap.c

```diff
@@ -78,8 +78,8 @@ struct distmap {
     struct ngpu_texture *texture;
     struct ngpu_rendertarget *rt;
     struct pgcraft *crafter;
-    struct block vert_block;
-    struct block frag_block;
+    struct ngpu_block_desc vert_block;
+    struct ngpu_block_desc frag_block;
```

Comment:
You're doing a bit more than just adding a `ngpu_` prefix

Reply:
Changes extracted to multiple commits before the move the ngpu directory. Should be in a better state. Sorry for the noise.